### ISSUE TITLE
Clarify consensus seeking process and quorum requirements

### ISSUE THREAD

**Issue by knolleary** [openjs-foundation/cross-project-council#204]
# tl;dr:

 1. Does an agenda item have to be on the list for at least 72 hours before the CPC is able to make a decision on it?
 2. If yes, should that be documented in the charter or some other document
 3. and if the CPC meeting has a quorum of voting members present (at least 50%), can it skip the 72 hour thing?

---

The charter defines the [decision making process](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-9-decision-making) as being Consensus Seeking. What is not clearly defined (in my reading of it) is how we determine that a suitable consensus has been reached.

The charter says (emphasis mine):

>When an agenda item has **appeared** to reach a consensus the moderator will ask "Does
anyone object?" as a final call for dissent from the consensus.

Can we better define what this means?

I think consensus means:

 - an agenda item has been open on the issue list for at least 72 hours (I _think_ I heard this suggested on the CPC call today... but it isn't in the charter and I may be mixing things up)
 - there are no unresolved dissenting opinions raised as comments on the issue
 - at a CPC meeting the item is discussed. No new dissenting opinion is raised when the moderator asks "Does anyone object?"

The question was also raised on today's call regarding what 'quorum' meant for the CPC. The [Voting section of the charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-10-voting) says:

> For all votes, a simple majority of all Voting CPC members for, or against, the issue wins. A Voting CPC member may choose to participate in any vote through abstention.

So for any decision that requires a vote (as a consensus has not been reached), quorum is defined as over 50% of Voting members present so that it is possible to achieve a simple majority.

If an item comes up in a CPC meeting that isn't explicitly an agenda item (some natural off-shoot of an agenda item discussion), can the CPC make a consensus driven decision there and then?  That would not meet the requirement of 72 hours cooling off on the issue list - but if there was quorum of voting members on the call, could a decision be made or would it need to be deferred to the next meeting?

**Comment by eemeli** [comment]
I think the 72-hour minimum should be encoded somewhere, and that it should be hard to break. Not impossible, just requiring actual effort and commitment. If something does get pushed through with a faster schedule, I'd argue that at the very least it should be re-examined and re-approved as the first item at the subsequent CPC meeting.

On the definition of consensus, I agree with the first two points but think that the last one is a bit misleading. Live CPC meetings are a means rather than an end, and should not be required for decisions to be made. Certainly having a live meeting with more than half the voting members present and not dissenting to an item is indicative of its support, but that should not be the only way that we get things done. For instance, if an issue has been open for a week and has been approved by a majority of voting members, would that not be similarly sufficient as an indicator of consensus? And extending further from there, having an issue open for two weeks while approved by a third would also indicate consensus, with a silent majority just not caring enough to participate.

**Comment by dmethvin** [comment]
My apologies for not making the meeting today. There was some confusion about the dial-in information and shortly thereafter I was pulled into a work meeting, which I went into thinking the meeting was cancelled.

I'm in agreement with @eemeli's points. In general I think people know when an issue might be controversial and require broader consensus, for example based on previous discussion of similar issues. As the CPC grows it will become difficult to get everyone together in real time. The 72-hour window is a good way to give people adequate notice and opportunity to participate without completely bringing the group to a halt in making decisions.

**Comment by mhdawson** [comment]
PR to clarify: https://github.com/openjs-foundation/cross-project-council/pull/205

**Comment by mhdawson** [comment]
Just to add my thoughts.  I'm aligned with what @eemeli has said.

My opinion is that decisions should still require an issue and the 72 hour waiting period. Just because there is consensus among those at a meeting and there are more than 50% of the voting members that does not mean other members should not still have the opportunity to raise objections and for there to be a discussion on those objections.

The voting members can always vote to override the process and land a PR with a decision anyway but that should be rare.

I think #205 reflects that so please review and make sure it covers any issues you think we need to clarify.

**Comment by mhdawson** [comment]
@knolleary #205 has landed do you think we've covered your question well enough for this to land?

**Comment by knolleary** [comment]
Thanks @mhdawson. All good.
